projects/fmcdaq2/src.mk

```diff
@@ -0,0 +1,51 @@
+################################################################################
+#									       #
+#     Shared variables:							       #
+#	- PROJECT							       #
+#	- DRIVERS							       #
+#	- INCLUDE							       #
+#	- PLATFORM_DRIVERS						       #
+#	- NO-OS								       #
+#									       #
+################################################################################
+
+# Uncomment to select the profile
+
+SRCS := $(PROJECT)/src/app/fmcdaq2.c					
+SRCS += $(DRIVERS)/axi_core/axi_adc_core/axi_adc_core.c			\
+	$(DRIVERS)/axi_core/axi_dac_core/axi_dac_core.c			\
+	$(DRIVERS)/axi_core/axi_dmac/axi_dmac.c				\
+	$(DRIVERS)/axi_core/clk_axi_clkgen/clk_axi_clkgen.c		\
+	$(DRIVERS)/axi_core/jesd204/axi_adxcvr.c			\
+	$(DRIVERS)/axi_core/jesd204/axi_jesd204_rx.c			\
+	$(DRIVERS)/axi_core/jesd204/axi_jesd204_tx.c			\
+	$(DRIVERS)/axi_core/jesd204/xilinx_transceiver.c		\
+	$(DRIVERS)/frequency/ad9523/ad9523.c				\
+	$(DRIVERS)/adc/ad9680/ad9680.c					\
+	$(DRIVERS)/dac/ad9144/ad9144.c					\
+	$(NO-OS)/util/util.c
+SRCS +=	$(PLATFORM_DRIVERS)/axi_io.c					\
+	$(PLATFORM_DRIVERS)/spi.c					\
+	$(PLATFORM_DRIVERS)/gpio.c					\
+	$(PLATFORM_DRIVERS)/delay.c
```

Comment:
Is it good practice to have all the `SRCS=` or can we condense them all into one?

Reply:
The pattern is kept similar to the other projects' makefiles.
The `srcs` is somehow splitted based on the modules that are included (in this case, based on the env vars `DRIVERS` for device drivers and `PLATFORM_DRIVERS` for platform specific drivers).